Update documentation for signup() (#757)
wasp-lang/wasp: web/docs/language/features.md

- Signup and Login forms at `@wasp/auth/forms/Signup` and `@wasp/auth/forms/Login` routes
- `logout` function
- `useAuth()` React hook

> It should be noted that the Signup form automatically logs in the user after signup.

Reviewer: ```suggestion
**NOTE:** If the signup is successful, the Signup form will automatically log in the user.
```
Just to follow the style we use in the rest of the docs :)

Author: Excellent.  :+1:

Author: Wow, I just saw that I can commit the suggestion from the UI.  :exploding_head:

Reviewer: Yeah, pretty cool. I've only learned about it recently too.

### `signup()`

> An action for signing up the user. This action does not login a user, you still need to call `login()`.

Author: Learned something new today.  Thanks!
